H_ecto_models.md

```diff
@@ -561,20 +567,17 @@ defmodule HelloPhoenix.User do
 end
 ```
 
-And here is how we would declare the complementary `belongs_to` relationship in `web/models/video.ex`:
+Since we used generator to scaffold our `Video` model and specified user association with `user_id:references:users` `belongs_to` relationship will already be defined for us in `web/models/video.ex`:
```

Comment:
When following the guide I was trying to add `belongs_to` to `web/models/video.ex` myself and didn't even notice that `belongs_to` was already there (thanks to generator). That's the reason I changed the text. @lancehalvorsen any comments/suggestions?